test/e2e/network_policy_helpers.go

```
@@ -0,0 +1,965 @@
+		g.Fail(fmt.Sprintf("%s/%s: expected podSelector %s=%s, got %v", policy.Namespace, policy.Name, key, value, policy.Spec.PodSelector.MatchLabels))
+	}
+}
+
+func requireIngressPort(policy *networkingv1.NetworkPolicy, protocol corev1.Protocol, port int32) {
+	g.GinkgoHelper()
+	if !hasPortInIngress(policy.Spec.Ingress, protocol, port) {
+		g.Fail(fmt.Sprintf("%s/%s: expected ingress port %s/%d", policy.Namespace, policy.Name, protocol, port))
+	}
+}
+
+func requireIngressFromNamespace(policy *networkingv1.NetworkPolicy, port int32, namespace string) {
```

golangci-lint (2.11.4)

[error] func requireIngressFromNamespace is unused

(unused)